global-test.properties

```diff
@@ -31,9 +31,8 @@ intersmash.activemq.operators.index_image=quay.io/jbossqe-eap/intersmash-activem
 intersmash.activemq.operators.package_manifest=activemq-artemis-operator
 intersmash.activemq.operators.channel=upstream
 
-intersmash.keycloak.realm_import.image=quay.io/keycloak/keycloak:21.1.1
+intersmash.keycloak.realm_import.image=quay.io/keycloak/keycloak:latest
```

Comment:
why `latest` when there is `22.0.4` in CRDs?

Reply:
`22.0.4` is actually the latest, if I don't go wrong. 
Do you mean that we should be explicitly referencing the tag here? 
My reasoning is to keep latest for images and catch up by updating CRDs, but your proposal could be good as well. Maybe we could fix it in a separate PR in case we decide to go this way, since the same applies for other images, I guess.
WDYT?

Reply:
See #99